.travis.yml

```diff
@@ -0,0 +1,30 @@
+language: python
+
+dist: bionic
+sudo: false
+
+cache: pip
+
+before_cache:
+  - rm -f $HOME/.cache/pip/log/debug.log
+
+# Don't run tests on WIP branches
+branches:
+  except:
+    - /^WIP-.*$/
+
+addons:
+  apt:
+    packages:
+    - libacl1-dev
+    - librsync-dev
+    - rdiff
+
+install:
+  - pip install -U pip tox virtualenv
+
+matrix:
+  include:
+    - python: 3.6
```

Comment:
Why Python 3.6? It seemed to me our reference implementation is 3.7.

Reply:
I had 3.6 because that was what Ubuntu 18.04 ships, but sure, let's keep 3.7 as our main target and use it for flake8 as well.